contracts/governance/Governor.sol

```diff
@@ -333,59 +336,133 @@ abstract contract Governor is Context, ERC165, EIP712, Nonces, IGovernor, IERC72
     }
 
     /**
-     * @dev See {IGovernor-execute}.
+     * @dev See {IGovernor-queue}.
      */
-    function execute(
+    function queue(
         address[] memory targets,
         uint256[] memory values,
         bytes[] memory calldatas,
         bytes32 descriptionHash
-    ) public payable virtual override returns (uint256) {
+    ) public virtual override returns (uint256) {
+        return _queue(targets, values, calldatas, descriptionHash);
```

Comment:
Honestly I'm very tempted to say we should merge `queue`+`_queue` and `execute`+`_execute`... I understand the "logic", but it feels wrong to have this separation if we don't use it for anything, it just adds lines of code and potential confusion.

Reply:
It also adds some consistency.

My concern is, if we merge them, and we later realize that for some reason we should split them, is that something we can do in a non breaking (minot) version ?

Reply:
The only concern, if we do that, is that overrides to the public function would not be present for the internal function, and depending on the logic they might have to.

I think it's a fairly small risk of that causing any issues for these functions. I think I'm comfortable documenting via changelog if we ever do that.

Reply:
It's not really clear in my mind why would we need to have the 3 of the functions. I agree with the rationale you shared and I think it makes sense but having a `_doExecute` is something we can add in a minor version if I'm not mistaken. Please correct me if wrong.

My preference would be having only `execute` and `_execute` but not super strong. I'll be open to it if it's clearer for the user. An example of clarity is the `_update` and `_updateWithAcceptanceCheck` in ERC721. It shows clearly what exactly each one of those do.

Reply:
It's the other way around. We need "`_doExecute`" now, it's used by the timelock extensions. What we don't know is about the separation between `execute` and `_execute`.

Reply:
I know it's the other way around, but the `_doExecute` can be `_execute` and the original `_execute` can be replaced with `_executeWithValidation`. Just putting an example, please don't discard my idea just because you interpreted differently.

Reply:
I'm not sure I understood the idea yet. Initially you said you prefer only `execute` and `_execute`, I assume you meant the current `execute` and `_doExecute` functions (the latter renamed to `_execute`). Would `_executeWithValidation` be a 3rd function with the semantics of the current `_execute`?

Reply:
> My preference would be having only execute and _execute but not super strong.

Because my opinion is not super strong, I'm open to having 3 if we change the names. I put the ERC721's case as an example.

Reply:
Ok, so I think we are aligned. We both prefer having only two functions, and if there are three functions they need to have better names.

`execute` and `_execute` (in the current PR) do exactly the same, so they shouldn't be named too differently, and similarly `_doExecute` does something different, so it should not be named `_execute`. If we want a better name we should find one for `_doExecute`, regardless of whether we keep `_execute` or not.

Reply:
Okay, since `_do{Queue,Execute}` was not great naming, I've renamed to `_{queue,execute}Operations`. I realize this is closer to the original name of `_{queue,execute}Calls`, but I think "operations" is better ("calls" is confusing to me because it's also a verb).

---

I would still remove `_queue` and `_execute` but I'll let @Amxx do it if he agrees. It's as simple as removing a few lines so that the body is moved to the public functions, since they're not even tested directly.